components/EditConnectedAccount.js

```diff
@@ -119,9 +119,10 @@ class EditConnectedAccount extends React.Component {
             </Button>
           </div>
         )}
-        {connectedAccount && connectedAccount.service === 'twitter' && (
-          <EditTwitterAccount collective={collective} connectedAccount={connectedAccount} />
-        )}
+        {connectedAccount &&
+          connectedAccount.service === 'twitter' &&
+          collective.type ===
+            'ORGANIZATION'(<EditTwitterAccount collective={collective} connectedAccount={connectedAccount} />)}
```

Comment:
Can you check this line? Looks weird.

Reply:
Fixed in https://github.com/opencollective/opencollective-frontend/pull/2753/commits/3dcc4737e7d48e802d9a071d7f312f575fd4f060